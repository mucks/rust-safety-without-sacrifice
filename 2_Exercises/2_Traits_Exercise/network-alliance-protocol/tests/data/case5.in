# Mixed workflow
STATE ForestCity 600
CITIZEN ForestCity 1 Alice Builder
CITIZEN ForestCity 2 Alex Researcher
CITIZEN ForestCity 3 Aria Validator
STATE Prospera 900
CITIZEN Prospera 4 Pax Builder
CITIZEN Prospera 5 Pia Researcher
CITIZEN Prospera 6 Pam Validator
CITIZEN Prospera 7 Phil Governor
CITIZEN Prospera 8 Pedro Builder
CITIZEN Prospera 9 Priya Researcher
ALLY ForestCity Zuitzerland
REPORT ForestCity
ALLIANCE_NEW GreenNet
ALLIANCE_ADD GreenNet ForestCity
ALLIANCE_ADD GreenNet Prospera
ALLIANCE_DESCRIBE GreenNet
POLY_SUMMARIES
