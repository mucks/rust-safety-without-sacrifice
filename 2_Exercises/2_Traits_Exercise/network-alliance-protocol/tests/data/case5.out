REPORT ForestCity: pop=3 gdp=600 per_capita=200 allies=1
ALLIANCE GreenNet: members=2 total_pop=9 avg_gdp=750
[State] ForestCity: pop=3 gdp=600
[Alliance] GreenNet: members=2 total_pop=9
